src/main/java/org/sagebionetworks/bridge/spring/controllers/CRCController.java

```diff
@@ -324,33 +350,39 @@ void httpBasicAuthentication() {
         // Verify the password
         Account account = accountService.authenticate(app, signIn);
 
-        // This method of verification entirely sidesteps RequestContext 
-        // initialization. Set up anything that is needed in this controller.
-        BridgeUtils.setRequestContext(new RequestContext.Builder()
-                .withCallerAppId(appId)
-                .withCallerSubstudies(BridgeUtils.collectSubstudyIds(account))
-                .build());
+        // This method of verification sidesteps RequestContext initialization
+        // through a session. Set up what is needed in the controller.
+        Set<String> substudies = BridgeUtils.collectSubstudyIds(account);
+        
+        RequestContext.Builder builder = new RequestContext.Builder().withCallerAppId(appId);
+        // Sage account can see all participants, others will only have access to their 
+        // substudy participants.
+        if (!substudies.contains("sage")) {
+            builder.withCallerSubstudies(substudies);
+        }
```

Comment:
This breaks our existing substudy model, and will cause other APIs to behave abnormally. Is this what we want?

Reply:
Changed to use an email credential rather than an external ID, so Sage accounts can be unbound from a substudy. This gives these accounts access to Columbia's external ID accounts.